cursor.cpp

```diff
@@ -0,0 +1,134 @@
+/**
+* If not stated otherwise in this file or this component's LICENSE
+* file the following copyright and licenses apply:
+*
+* Copyright 2022 RDK Management
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+#include "cursor.h"
+#include "rdkshell.h"
+#include "rdkshellimage.h"
+#include "essosinstance.h"
+#include "logger.h"
+
+
+namespace RdkShell
+{
+    const uint32_t DEFAULT_CURSOR_SIZE = 64;
+    const double DEFAULT_INACTIVITY_DURATION = 5.0;
+
+    Cursor::Cursor(const std::string& fileName)
+        : mIsLoaded(false)
+        , mX(0), mY(0)
+        , mWidth(DEFAULT_CURSOR_SIZE), mHeight(DEFAULT_CURSOR_SIZE)
+        , mInactivityDuration(DEFAULT_INACTIVITY_DURATION)
+        , mOffsetX(0), mOffsetY(0)
+        , mLastUpdateTime(0.0)
+        , mIsVisible(true)
```

Comment:
Can you please make the cursor not visible by default?

Reply:
done